docs/sources/flow/_index.md

```diff
@@ -94,5 +105,8 @@ This feature is experimental, and it doesn't support all River components.
 [Tutorials]: "/docs/agent/ -> /docs/agent/<AGENT_VERSION>/flow/tutorials/"
 [Tutorials]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/send-data/agent/flow/tutorials/
 [Reference]: "/docs/agent/ -> /docs/agent/<AGENT_VERSION>/flow/reference/"
-[Reference]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/send-data/agent/flow/reference/
+[Reference]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/send-data/agent/flow/reference/"
+[components]: "/docs/agent/ -> /docs/agent/<AGENT_VERSION>/flow/reference/"
+[components]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/send-data/agent/flow/reference/components"
+[distribution]: "https://opentelemetry.io/ecosystem/distributions/"
```

Comment:
Move this link definition out of the `docs/reference` block (and when you do, remove the double quotes around the URL). The link definition can go above or below this block... I usually put them above.